### DISCUSSION TITLE
Can't connect to github repository in intellij

### DISCUSSION THREAD

**Discussion by nina985** [community/community#138367] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

When i check the pull request tab i see the message: _Could not connect to repository (repository name, it's a repo within an organization) with account (my account) Request response: Access to this site has been restricted. If you believe this is an error, please contact https://support.github.com. Log in again…_
I am logged in with my git account in intellij and the admin says that my access isn't restricted. I'm not the most familiar with github, up until now i've only ever worked with it for individual assignments, so my guess is that i'm doing something wrong, but i have no idea what :,)
Thanks in advance for any advice

**Comment by SandithKariyawasam** [comment]
Even though your admin mentioned that your access isn’t restricted, it’s still a good idea to double-check your permissions. Make sure you have the correct level of access, like being able to read or write to the repository. You can usually find this information in the repository settings under 'Collaborators' or 'Teams' to confirm everything’s set up right.
